SVGImageView.h

```diff
@@ -1,14 +1,10 @@
-#import <TargetConditionals.h>
+#import "SVGPortability.h"
 
-// This setup is just so Interface Builder doesn't get confused.
```

Comment:
Looks good to me: 

![image](https://cloud.githubusercontent.com/assets/1756909/17481264/6c9f526c-5d7c-11e6-8e86-56606e94955a.png)

Is this the expected behaviour?